docs/intro.rst

```diff
@@ -53,7 +53,7 @@ Pywbem provides the following Python APIs:
   An API for compiling MOF files or strings into a CIM repository (e.g. on a
   WBEM server), or for test-compiling MOF.
 
-* :ref:`Mock support` -
+* :ref:`Mock WBEM server` -
```

Comment:
Change that term like this also in the README.rst file.

Reply:
DONE